command/status.go

```diff
@@ -0,0 +1,135 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+type StatusCommand struct {
+	Meta
+}
+
+func (c *StatusCommand) Run(args []string) int {
+	// Get the HTTP client
+	client, err := c.Meta.Client()
+	if err != nil {
```

Comment:
Think you want to do this after parsing. The parsed flags inform the client how to set up the addresses/tls/etc.